tests/test_integrated_channels/test_canvas/test_client.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for clients in integrated_channels.
+"""
+
+import datetime
+import unittest
+import pytest
+
+from six.moves.urllib.parse import urljoin  # pylint: disable=import-error
+from freezegun import freeze_time
+
+from django.utils import timezone
+
+from integrated_channels.canvas.client import CanvasAPIClient
+from test_utils import canvas_factories
+
+NOW = datetime.datetime(2017, 1, 2, 3, 4, 5, tzinfo=timezone.utc)
+NOW_TIMESTAMP_FORMATTED = NOW.strftime('%F')
+
+
+@freeze_time(NOW)
+@pytest.mark.django_db
+class TestCanvasApiClient(unittest.TestCase):
```

Comment:
This test is also just a 'shell' right now to ensure the client itself is correctly instantiated, but much of the testing here will be done in the subtask to implement this client fully, coming up